estate/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from odoo import fields, models
+
+
+class Estate(models.Model):
+    _name = "estate.property"
+    _description = "Properties of estate entities."
+
+    name = fields.Char(required=True)
```

Comment:
```suggestion
    name = fields.Char(string="Name", required=True)
```
It's a good practice to add an explicit `string=` to fields that are meant to be in the view, else you let Odoo decide what he will use, which is a bit random.